src/Shared/LayoutRenderers/AspNetRequestCookieLayoutRenderer.cs

```diff
@@ -111,10 +116,14 @@ private IEnumerable<string> GetCookieNames(Cookies cookies)
             }
         }
 #else
-        private IEnumerable<KeyValuePair<string, string>> GetCookies(IRequestCookieCollection cookies, IEnumerable<string> cookieNames)
+        private IEnumerable<KeyValuePair<string, string>> GetCookieValues(IRequestCookieCollection cookies, bool excludeKeys)
```

Comment:
Minor, I prefer the same terms (names instead of keys)

Reply:
Think your mobile phone is cheating. I have moved the lookup of `cookieNames `into the `GetCookieValues`-method. It is a new parameter called `excludeKeys`

Reply:
We have now `CookieNames` and `excludeKeys`. It should be `excludeNames` IMO (we can't change it to CookieKeys as that's a breaking change)

Reply:
Think you are still not seeing the source code correctly but cheated by your mobile

Reply:
Don't you see the CookieNames property? And the excludeKeys parameter name?

Reply:
Yes and cookieNames is now a variable inside the function (it is not gone, but NOT passed as parameter), and `excludeKeys` is passed as parameter to reuse the check for both NLog.Web and Nlog.Web.AspNetCore. Again you are not seeing the code, but just the diff on your phone.

Reply:
No it not gone. My point is that name=key. So 2 terms for the same thing.

Reply:
So excludeNames or excludeCookieNames

Reply:
Ok renamed from `excludeKeys` to `checkForExclude`. No names. No keys.

Reply:
I was confused by your sentence "we can't change it to CookieKeys as that's a breaking change".

Reply:
When I now see it, I guess I was also confused ;)

Anyway, perfect now! Thanks!